src/attributes/utils/data.ts

```diff
@@ -104,3 +121,38 @@ export const getAttributesAfterFileAttributesUpdate = (
 
   return uploadedFileAttributes.concat(removedFileAttributes);
 };
+
+export const getAttributesDisplayData = (
+  attributes: AttributeInput[],
+  attributesWithNewFileValue: FormsetData<null, File>,
+  referencePages: SearchPages_search_edges_node[]
+) =>
+  attributes.map(attribute => {
+    if (attribute.data.inputType === AttributeInputTypeEnum.REFERENCE) {
+      return {
```

Comment:
Would like to see content of this return as function `getParsedReferenceData` or something so this function is clearer in what it does